jobs/submanSUSEBuildJob.groovy

```diff
@@ -0,0 +1,57 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.submanJobFolder
+String githubOrg = binding.variables['CANDLEPIN_JENKINS_GITHUB_ORG'] ?: 'candlepin'
+
+def DISTROS = [ // values can be found via `osc repos`
+    'opensuse42': 'openSUSE_Leap_42.2',
+    'sles11': 'SUSE_SLE-11_SP3_standard',
+    'sles12': 'SLE_12_SP1',
+]
+
+DISTROS.each { name, repo_name ->
+    def rhsmJob = job("$baseFolder/subscription-manager-suse-build-${name}"){
+        description("Build SUSE RPMs locally for ${name}")
+        label(name)
+        parameters {
+            // FIXME move out of home:kahowell?
+            stringParam('obs_project_name', 'home:kahowell', 'Build Service project to reference for config.')
```

Comment:
Okay, cool.  I thought it was a local reference to a path.  No complaint here now.